### PR TITLE
Add command line example for cloning ProjectScaffold repo

### PR DESCRIPTION
**Pull request by mexx**

### CODE REVIEW

**Comment by forki** [comment]
I think instead of cloning we should suggest to:

* clone
* delete ".git" folder
* git init

This is what I do and it works really well. @mexx could you please rewrite your guide for this workflow? Maybe as a subpage?

File: docs/content/index.md [hunk 1]
```diff
@@ -15,9 +15,21 @@ This includes [Paket](paket-package-management.html) dependency management and a
...
+### Getting the project scaffold
+
+This first thing to do is to copy the ProjectScaffold repository to your own workspace.
+Download the [archive](https://github.com/fsprojects/ProjectScaffold/archive/master.zip) or use following script:
+
+    // on windows in target directory
+    $ git clone --depth 1 https://github.com/fsprojects/ProjectScaffold.git .
+    $ rd /S /Q .git
+    $ git init
+
```
Comment (mexx) [inline docs/content/index.md]: @forki like this one?

**Comment by forki** [comment]
ah you already did? I didn't notice. I wonder if we can make this part of the init process.

**Comment by pblasucci** [comment]
@forki This is done now (pending merge). See PR #197

**Pull request closed without merging**